components/edit-collective/EditHostInvoice.js

```diff
@@ -43,33 +51,44 @@ const EditHostInvoice = ({ collective }) => {
           {getErrorFromGraphqlException(setValue).message}
         </MessageBox>
       )}
-      <Flex flexWrap="wrap">
+      <Flex flexWrap="wrap" flexDirection="column">
         <StyledInput
           placeholder="Payment Receipt"
           defaultValue={value}
           onChange={e => setValue(e.target.value)}
           width="100%"
           maxWidth={300}
+          my={3}
+        />
+
+        <StyledTextarea
+          placeholder="Add any other text to appear on payment receipts, such as your organization's tax ID number, info about tax deductibility of contributions, or a custom thank you message."
```

Comment:
As per https://docs.opencollective.com/help/developers/guidelines, this should be internationalized.